packages/migrate/src/utils/setupMSSQL.ts

```diff
@@ -25,40 +25,57 @@ function getMSSQLConfig(url: string): mssql.config {
   }
 }
 
-export async function setupMSSQL(options: SetupParams): Promise<void> {
+export async function setupMSSQL(options: SetupParams, databaseName: string): Promise<void> {
   const { connectionString } = options
   const { dirname } = options
   const config = getMSSQLConfig(connectionString)
   const connectionPool = new mssql.ConnectionPool(config)
   const connection = await connectionPool.connect()
 
   try {
-    await connection.query(`
-      CREATE DATABASE [tests-migrate-shadowdb]
-      CREATE DATABASE [tests-migrate]
+    if (databaseName === 'tests-migrate') {
+      await connection.query(`
+CREATE DATABASE [tests-migrate-shadowdb]
+CREATE DATABASE [tests-migrate]
+    `)
+    } else {
+      await connection.query(`
+CREATE DATABASE [${databaseName}]
```

Comment:
What is this new case used for?

Reply:
I wanted to create a separate database to avoid concurrency issues during tests here